### ISSUE TITLE
A Group Participant who is promoted to Admin has no option to leave the Group. 

### ISSUE THREAD

**Issue by JohnELester** [urbit/urbit#3957]
**Describe the bug**
A Group Participant who is promoted to Admin will no longer be given the option to leave the group.  

**To Reproduce**
1. As a default Participant in a Group, you see the option to Leave the group on the Group Settings Page.
![leave group](https://user-images.githubusercontent.com/25461543/99595398-a72bec80-29c2-11eb-8ec5-c31af08db68b.png)

2. The Group Owner promotes the Participant to Admin on the Participants Page.

3. The Participant now only sees the option to Delete the group with no option to just leave the Group.
![delete group](https://user-images.githubusercontent.com/25461543/99595643-143f8200-29c3-11eb-8a6d-dcadbf09363c.png)

**Expected behavior**
A Participant should be able to leave a Group if they are an Admin. 

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome, Firefox, Brave
 - Base hash of your urbit ship: tqdqb

**Comment by tacryt-socryp** [comment]
@urcades What are your thoughts on this? Should an admin have to be demoted from admin position to be removed? Can we ensure we don't get into a situation in which there are no admins?

**Comment by urcades** [comment]
I've actually been working on a "permissions and roles" matrix that I wanted to get cleaned up and attached to our current working 1.6 ticket about permissions — here's the current state:

https://docs.google.com/spreadsheets/d/149cAIT7dhFCGlex_5wEuXcOsQmTyIoMEkH5591S1AB8/edit?usp=sharing

tl;dr: the proposal I'm trying to come to bearing with is a distinction between a "Group Owner" (a superadmin, basically) and a "regular admin", who maintains nearly completely elevated capabilities besides crucial ones like deleting the group.

I'm thinking that at any one time, there should only be one "Group Owner" who effectively hosts the group and keeps it online, and this should be the only person who cannot leave the group. Admins should definitely be able to leave a group, I believe.

**Comment by urcades** [comment]
also, mentioning you so you know I responded to you in particular @tacryt-socryp ! 

Please lmk what you think about the above. I was going to get it reviewed by others as a sanity check.

**Comment by matildepark** [comment]
Progress is currently being tracked in https://github.com/urbit/landscape/issues/197.
